api/src/permission-configs/permission_policy.csv

```diff
@@ -11,10 +11,12 @@ p, anonymous, user, true, create
 
 p, admin, asset, true, .*
 p, jurisdictionAdmin, asset, true, .*
+p, limitedJurisdictionAdmin, asset, true, .*
 p, partner, asset, true, .*
 
 p, admin, multiselectQuestion, true, .*
 p, jurisdictionAdmin, multiselectQuestion, true, .*
+p, limitedJurisdictionAdmin, multiselectQuestion, true, .*
 p, partner, multiselectQuestion, true, .*
 
 p, admin, applicationMethod, true, .*
```

Comment:
I think this should be true for the new role

Reply:
I am able to add/change applicationMethod. There is no calls using canOrThrow with this type. Can probably be removed going forward.